bin/fix-dbs-parentage

```diff
@@ -58,29 +62,29 @@ def getMissingParentSummaryUsingBlock(datasetList, fileFix=False, insertFlag=Fal
 
     if insertFlag:
```

Comment:
If one doesn't pass `insertFlag`, it will run this method all the way until the insert and fix methods below.

Reply:
Yes but with insertFlag = False. which run the those method without actually inserting to dbs. This is just a test case to check how long it takes and correctness of the parentage.